### ISSUE TITLE
release/public-v1: Fix legacy Fortran extension

### ISSUE THREAD

**Issue by DusanJovic-NOAA** [NOAA-EMC/UFS_UTILS#61]
```
/home/dusan/simple-ufs/src/preproc/sorc/chgres_cube.fd/write_data.F90:1167:31:

 1167 |    WRITE(OUTFILE, '(A, I1, A)'), 'out.atm.tile', tile, '.nc'
      |                               1
Warning: Legacy Extension: Comma before i/o item list at (1)
/home/dusan/simple-ufs/src/preproc/sorc/chgres_cube.fd/write_data.F90:1637:33:

 1637 |      WRITE(OUTFILE, '(A, I1, A)'), 'out.sfc.tile', tile, '.nc'
      |                                 1
Warning: Legacy Extension: Comma before i/o item list at (1)
[100%] Linking Fortran executable chgres_cube.exe
```

**Comment by GeorgeGayno-NOAA** [comment]
Dusan, I can add this small change to one of my other issues and then merge to 'develop'.  To get this in the public release branch, you will need to contact the branch code manager.

**Comment by GeorgeGayno-NOAA** [comment]
What compiler is giving this warning?

**Comment by climbfuji** [comment]
GNU does.

**Comment by DusanJovic-NOAA** [comment]
GNU (gfortran)

> What compiler is giving this warning?

GNU (gfortran)

**Comment by GeorgeGayno-NOAA** [comment]
Merged to 'develop' at 72c636c.  For convenience, I added this small fix to issue #64.

**Comment by DusanJovic-NOAA** [comment]
Who is the code manager of release/public-v1 branch?
